lib/LMSManagers/LMSDivisionManager.php

```diff
@@ -176,10 +183,23 @@ public function UpdateDivision($division)
 			inv_paytype=?, email=?, description=?, status=?, tax_office_code = ?
 			WHERE id=?', array_values($args));
 
+        if (!empty($division['diff_users_del'])) {
+            foreach ($division['diff_users_del'] as $userdelid) {
+                $this->db->Execute('DELETE FROM userdivisions WHERE userid = ? AND divisionid = ?', array($userdelid, $division['id']));
+            }
+        }
+
+        if (!empty($division['diff_users_add'])) {
+            foreach ($division['diff_users_add'] as $useraddid) {
+                $this->db->Execute('INSERT INTO userdivisions (userid, divisionid) VALUES(?, ?)', array($useraddid, $division['id']));
+            }
+        }
+
         $lm = new LMSLocationManager($this->db, $this->auth, $this->cache, $this->syslog);
         $lm->UpdateAddress($division);
 
         if ($this->syslog) {
+            $args['users'] = implode(',', $division['users']);
             $this->syslog->AddMessage(SYSLOG::RES_DIV, SYSLOG::OPER_UPDATE, $args);
         }
```

Comment:
Zapisywać w SYSLOG tych co usunąłeś i tych co dodałeś.
Piszesz dokładnie to co robisz na bazie danych.